Error while building example: takeoff_and_land

Hi Julian, I did build DroneLink as mentioned. I wanted to try running example. Got below error.
Could you please check ?
```
~/DroneLink/example$ make
[  3%] Built target gtest
[  6%] Built target gtest_main
[ 56%] Built target dronelink
[ 57%] Building CXX object CMakeFiles/integration_tests_runner.dir/integration_tests/curl.cpp.o
/home/shakthi/DroneLink/integration_tests/curl.cpp:10:25: error: ‘testing’ has not been declared
 class CurlTest : public testing::Test
                         ^
/home/shakthi/DroneLink/integration_tests/curl.cpp:10:34: error: expected ‘{’ before ‘Test’
 class CurlTest : public testing::Test
                                  ^
/home/shakthi/DroneLink/integration_tests/curl.cpp:12:1: error: expected primary-expression before ‘protected’
 protected:
 ^
/home/shakthi/DroneLink/integration_tests/curl.cpp:12:1: error: expected ‘}’ before ‘protected’
/home/shakthi/DroneLink/integration_tests/curl.cpp:12:1: error: expected ‘,’ or ‘;’ before ‘protected’
/home/shakthi/DroneLink/integration_tests/curl.cpp:18:24: error: ‘virtual’ outside class declaration
     virtual void SetUp()
                        ^
/home/shakthi/DroneLink/integration_tests/curl.cpp: In function ‘void SetUp()’:
/home/shakthi/DroneLink/integration_tests/curl.cpp:20:15: error: ‘clean’ was not declared in this scope
         clean();
               ^
/home/shakthi/DroneLink/integration_tests/curl.cpp:21:9: error: ‘_file_url_existing_http’ was not declared in this scope
         _file_url_existing_http =
         ^
/home/shakthi/DroneLink/integration_tests/curl.cpp: At global scope:
/home/shakthi/DroneLink/integration_tests/curl.cpp:29:27: error: ‘virtual’ outside class declaration
     virtual void TearDown()
                           ^
/home/shakthi/DroneLink/integration_tests/curl.cpp: In function ‘void TearDown()’:
/home/shakthi/DroneLink/integration_tests/curl.cpp:31:15: error: ‘clean’ was not declared in this scope
         clean();
               ^
/home/shakthi/DroneLink/integration_tests/curl.cpp: At global scope:
/home/shakthi/DroneLink/integration_tests/curl.cpp:45:1: error: expected declaration before ‘}’ token
 };
 ^
CMakeFiles/integration_tests_runner.dir/build.make:422: recipe for target 'CMakeFiles/integration_tests_runner.dir/integration_tests/curl.cpp.o' failed
make[2]: *** [CMakeFiles/integration_tests_runner.dir/integration_tests/curl.cpp.o] Error 1
CMakeFiles/Makefile2:69: recipe for target 'CMakeFiles/integration_tests_runner.dir/all' failed
make[1]: *** [CMakeFiles/integration_tests_runner.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
```

Thanks

--- a/example/CMakeLists.txt
+++ b/example/CMakeLists.txt
@@ -7,6 +7,9 @@ add_definitions("-std=c++11 -Wall -Wextra -Werror")
 # Add DEBUG define for Debug target
 set(CMAKE_CXX_FLAGS_DEBUG "-DDEBUG")
 
+# This finds thread libs on Linux, Mac, and Windows.
+find_package(Threads REQUIRED)
+
 include_directories(
     # Not needed if installed system-wide
     ../install/include
@@ -20,4 +23,5 @@ target_link_libraries(takeoff_and_land
     ${CMAKE_SOURCE_DIR}/../install/lib/libdronelink.a
     # If installed system-wide:
     # dronelink
+    ${CMAKE_THREAD_LIBS_INIT}
 )

